src/com/twitter/intellij/pants/service/project/PythonVenvBuilder.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package com.twitter.intellij.pants.service.project;
+
+import com.intellij.openapi.diagnostic.Logger;
+import com.twitter.intellij.pants.service.project.model.PythonInterpreterInfo;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+public class PythonVenvBuilder {
+
+  private final Path executable;
+
+  private static final String executableName = "venv_builder.pex";
+
+  protected static final Logger LOG = Logger.getInstance(PythonVenvBuilder.class);
+
+  private PythonVenvBuilder(Path executable) {
+    this.executable = executable;
+  }
+
+  public PythonInterpreterInfo build(String target, Path venvDir){
+    LOG.info(String.format("Invoking the .venv builder with target %s at %s", target, venvDir));
+    try {
+      Process process = new ProcessBuilder(executable.toString(),
+                                           "--target", target,
```

Comment:
don't we want to/can't we pass all the target specs specified during the import rather than one? Or we are doing it multiple times for one project (once. per target)?

Reply:
The venv_builder tool seems to support only one target argument, but I have found a way around this.